src/assets/src/components/queue.tsx

```diff
@@ -224,26 +238,24 @@ function QueueAttendingJoined(props: QueueAttendingProps) {
 
     // Card content
     const changeMeetingType = props.queue.my_meeting?.assignee
-        ? <small className="ml-2">(A Host has been assigned to this meeting. Meeting Type can no longer be changed.)</small>
+        ? <span className="ms-2 fs-6">(A Host has been assigned to this meeting. Meeting Type can no longer be changed.)</span>
```

Comment:
CSS associated with `small` seems to have changed, so I needed to use `fs-6` to maintain sizing of these hints in this file.